posts/views.py

```diff
@@ -86,6 +91,39 @@ def PostRepostView(request, pk=None):
     return redirect('posts:post', pk=repost.pk)
 
 
+def ParseContent(content):
+    hashtags = re.findall(r"#(\w+)", content)
+    mentions = re.findall(r"@(\w+)", content)
```

Comment:
This is the hashtag feature, please try next time to not include unrelated code in a PR.